README.md

```diff
@@ -48,11 +48,45 @@ secret key used to authenticate the session. Inside the handler, we call
 session values in session.Values, which is a `map[interface{}]interface{}`.
 And finally we call `session.Save()` to save the session in the response.
 
-Important Note: If you aren't using gorilla/mux, you need to wrap your handlers
-with
+## Avoiding Memory Leaks
+
+If you are not using gorilla/mux (or are using context), you will
+need to perform additional cleanup or else you will leak memory!
```

Comment:
This warning is too broad: using gorilla/context + mux should be fairly rare—and getting rarer, because gorilla/context doesn't play nice with Go 1.7+.

Reply:
however using gorilla/context + mux is not rare because mux.Vars() and this session library still uses gorilla context..so whoever uses mux or this library will be running into this issue